addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -300,52 +283,38 @@ options.registry.carousel = options.Class.extend({
      * @see this.selectClass for parameters
      */
     addSlide: function (previewMode) {
-        var self = this;
-        var cycle = this.$inner.find('.carousel-item').length;
-        var $active = this.$inner.find('.carousel-item.active, .carousel-item.prev, .carousel-item.next').first();
-        var index = $active.index();
-        this.$('.carousel-control-prev, .carousel-control-next, .carousel-indicators').removeClass('d-none');
-        this.$indicators.append('<li data-target="#' + this.id + '" data-slide-to="' + cycle + '"></li>');
-        var $clone = $active.clone(true);
-        $clone.removeClass('active').insertAfter($active);
-        _.defer(function () {
-            self.$target.carousel().carousel(++index);
-            self._rebindEvents();
-        });
+        this.$controls.removeClass('d-none');
+        this.$indicators.append($('<li>', {
+            'data-target': '#' + this.$target.attr('id'),
+            'data-slide-to': this.$items.length,
+        }));
+        // Need to remove editor data from the clone so it gets its own.
+        const $active = this.$items.filter('.active');
+        $active.clone(true)
+            .removeData('snippet-editor')
```

Comment:
Should not this simply be `.clone(false)` ? I don't know. This could probably be needed in < master so ? Maybe it is worth a separate fix then ?